src/services/usersServices.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { unicId } = require('../helpers/helpers');

```

Comment:
There is a typo in the function name, it should be 'uniqueId' instead of 'unicId'. Make sure to rename it in the helpers module as well.